e2e/test_bootstrap_git_url.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Test bootstrapping from a requirement with a git+https URL witout specifying a
+# version tag.
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$SCRIPTDIR/common.sh"
+
+GIT_REPO_URL="https://opendev.org/openstack/stevedore.git"
+
+fromager \
+  --debug \
+  --log-file="$OUTDIR/bootstrap.log" \
+  --error-log-file="$OUTDIR/fromager-errors.log" \
+  --sdists-repo="$OUTDIR/sdists-repo" \
+  --wheels-repo="$OUTDIR/wheels-repo" \
+  --work-dir="$OUTDIR/work-dir" \
+  --settings-dir="$SCRIPTDIR/changelog_settings" \
+  bootstrap "stevedore @ git+${GIT_REPO_URL}"
+
+find "$OUTDIR/wheels-repo/" -name '*.whl'
```

Comment:
I am not sure whether we need this since we are checking `EXPECTED_FILES` on line 25? I maybe wrong though

Reply:
Ah, I see! 
okay